### PR TITLE
Linting | Adding linting to all brands + improved documentation

### PR DESCRIPTION
**Pull request by elliottcrush**
#### What does this PR do? (please provide any background)

- [x] Adds default linting to all brands

- [x] Improves documentation around CI deployment

- [x] Fixes 3 compiling errors and version bumped accordingly

- [x] Get rid of that awful addition to the PR template

For the moment - the linting logs around only found from viewing the `npm run lint` task on CI for your pull request. Alternatively you can run that command locally to view the hell hole that is all the default warnings.

_This is not a defined ruleset we want to implement in make-up yet. We want to do a proof of concept before we consider sharing it with the wider team._

#### How can this be tested?
As mentioned above

#### What gif best describes how you feel about this work?
dog-turns-head-upside-down

- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h2>Brand Version Notifications</h2><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/seo-hx-core">SEO Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/email-hx-landing">Email Landing Pages</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapplite">Tripapp</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^2.0.3</td><td>❌</td></tr><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/render">Render</i></a> has the following brand dependencies:</h4><table><tr><td>HYBRID</td><td>4.0.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/trip-planner">Trip Planner</i></a> has the following brand dependencies:</h4><table><tr><td>AIRPARKS</td><td>^1.0.3</td><td>❌</td></tr><tr><td>HYBRID</td><td>4.0.5</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/hx-customer-contact">Manage My Booking</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>3.0.0</td><td>✅</td></tr></table><h4><i><a href="https://github.com/holidayextras/static-site-generator-sites/tree/master/system-maintenance">System Maintenance</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.3</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/tripapp-cordova">Mobile Booking App</i></a> has the following brand dependencies:</h4><table><tr><td>HX</td><td>^2.2.4</td><td>❌</td></tr></table><h4><i><a href="https://github.com/holidayextras/cordova-countdown">Mobile Countdown App</i></a> has the following brand dependencies:</h4><table></table><h4><i><a href="https://github.com/holidayextras/themeBlueprint">SB Platform</i></a> has the following brand dependencies:</h4><table></table><p><small><strong>Please consider these versions before merging</strong></small></p>

File: circle.yml [hunk 1]
```diff
@@ -3,13 +3,13 @@ machine:
...
     branch: master
     commands:
-    - echo -e "$NPM_USERNAME\n$NPM_PASSWORD\n$NPM_EMAIL" | npm login
-    - ./scripts/bash/publish.sh
-    - npm run theme-deployment
-    - ./scripts/bash/deploy.sh
+    - echo -e "$NPM_USERNAME\n$NPM_PASSWORD\n$NPM_EMAIL" | npm login # Use HX credentials to access our NPM account
+    - npm run deploy:npm # Publish brand changes to npmjs.com
+    - npm run theme-deployment # Compile theme files to local directory
+    - npm run deploy # Sync assets and stylesheets with S3 and invalidate Cloudfront
```
Comment (h0rhay) [inline circle.yml]: Why have we changed from the bash scripts?

**Pull request merged**